docker-compose.dev.yml

```diff
@@ -46,6 +49,7 @@ services:
       # NUMPROCS: 1
       CVAT_DEBUG_ENABLED: '${CVAT_DEBUG_ENABLED:-no}'
       CVAT_DEBUG_PORT: '9093'
+      COVERAGE_ENABLED: '${CVAT_DEBUG_WAIT_CLIENT:-no}'
```

Comment:
Can we use only `COVERAGE_PROCESS_START`? I mean if `COVERAGE_PROCESS_START` is defined then we assume that `COVERAGE_ENABLED = "yes"`. Do we really need both of these variables?

Reply:
No, we don't need both of these variables. I removed `COVERAGE_ENABLED`